spec/requests/api/merge_requests_spec.rb

```diff
@@ -377,12 +393,40 @@
 
   describe "GET :id/merge_request/:merge_request_id/comments" do
     it "should return merge_request comments" do
-      get api("/projects/#{project.id}/merge_request/#{merge_request.id}/comments", user)
-      expect(response.status).to eq(200)
-      expect(json_response).to be_an Array
-      expect(json_response.length).to eq(1)
-      expect(json_response.first['note']).to eq("a comment on a MR")
-      expect(json_response.first['author']['id']).to eq(user.id)
+      get api(merge_request_comments_url, user)
+      response.status.should == 200
+      json_response.should be_an Array
+      json_response.length.should == 1
+      json_response.first['note'].should == "a comment on a MR"
+      json_response.first['author']['id'].should == user.id
+      json_response.first['file_path'].should be_nil
+      json_response.first['line'].should be_nil
+    end
+
+    it "should return merge_request comments with line and path" do
+      line_code = Gitlab::Diff::LineCode.generate('files/ruby/feature.rb', 5, 0)
+      create(:note_on_merge_request,
+             author: user,
+             project: project,
+             noteable: merge_request,
+             note: "comment with line",
+             line_code: line_code
+            )
+
+      get api(merge_request_comments_url, user)
+      response.status.should == 200
+      json_response.should be_an Array
+      json_response.length.should == 2
+      note =
+        if json_response.first['note'] == "comment with line"
+          json_response.first
+        else
+          json_response.last
+        end
+      note["note"].should == "comment with line"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.